src/scenes/Root/Creates/CreateMenu.tsx

```diff
@@ -46,15 +45,17 @@ export const CreateButtonMenu = (props: CreateButtonMenuProps) => {
 
   return (
     <>
-      <CreateButton
+      <Button
+        variant="contained"
+        color="error"
```

Comment:
The big reason why I had a CreateButton component was to decouple the "create action" from the error palette. Those are kinda two different things entirely. Now what we could do is add a custom palette that would allow us to do `color="create"`. MUI has docs for this. 

That said it's possible we don't even like this style or maybe this is the only place it happens. In which case I'd say don't worry about it.